tests/dummy/config/environment.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
 
 module.exports = function (environment) {
-  var ENV = {
+  const ENV = {
```

Comment:
Should remain `var` to remain consistent with all the other `ember-cli 2.12.3` installations.